piano-hero/mobile/src/main/java/com/novoda/pianohero/AndroidThingsActivity.java

```diff
@@ -7,9 +7,27 @@
 
 public class AndroidThingsActivity extends AppCompatActivity {
 
+    private MidiKeyboardDriver midiKeyboardDriver;
+
     @Override
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         Log.d("!!!", "I'm running");
+
+        midiKeyboardDriver = new MidiKeyboardDriver.KeyStationMini32(this);
```

Comment:
I coded it for my keyboard and it works so I named it so `KeyStationMini32` I don't have another keyboard to see if it'll work for another :-) but it shouldn't be hard to make any midikeyboard implement MidiKeyboardDriver